sympy/external/tests/test_sage.py

```diff
@@ -222,6 +222,17 @@ def test_undefined_function():
     assert bool(sf(sx) == f(x)._sage_())
     #assert bool(f == sympy.sympify(sf))
 
+def test_abstract_function():
+    x,y = sympy.symbols('x y')
+    f = sympy.Function('f')
+    expr =  f(x,y)
+    sexpr = expr._sage_()
+    invexpr = sexpr._sympy_()
+    assert type(sexpr) is sage.symbolic.expression.Expression, "converted expression %r is not sage expression" % sexpr
```

Comment:
I think `assert isinstance()` is probably a more standard way to do this kind of check.